### DISCUSSION TITLE
Using an object instead of an array.

### DISCUSSION THREAD

**Discussion by bootstrapuser** [PhilippeMarcMeyer/FieldsLinker#32] [General]
Hi, How are you?
I just found your plugin, it's great! congratulations.. and the functionally it's exactly what I have been looking for. Would it be possible to use 2 objects instead of 2 arrays? This is because I should only save the id in the database but show the name of the item in the list.
Hoping to hear soon from you, thanks!

**Comment by PhilippeMarcMeyer** [comment]
Hello,
I had not any time today for that
What are you going to do with the object ? Serialise it to store it ? Is it
why you need an id : a PK ?
Could you do a conversion from and to an array ?
Have you an example to show ?

Sincerely

Philip




Le jeu. 27 juil. 2023, 22:22, bootstrapuser ***@***.***> a
écrit :
